CONTRIBUTING.md

```diff
@@ -70,53 +109,62 @@ Our pre-commit hooks verify that your commit message matches this format when co
 
 ### Linting and tests
 
-[ESLint](https://eslint.org/), [Prettier](https://prettier.io/), [TypeScript](https://www.typescriptlang.org/)
+[ESLint](https://eslint.org/), [Prettier](https://prettier.io/),
+[TypeScript](https://www.typescriptlang.org/)
 
-We use [TypeScript](https://www.typescriptlang.org/) for type checking, [ESLint](https://eslint.org/) with [Prettier](https://prettier.io/) for linting and formatting the code, and [Jest](https://jestjs.io/) for testing.
+We use [TypeScript](https://www.typescriptlang.org/) for type checking,
+[ESLint](https://eslint.org/) with [Prettier](https://prettier.io/) for linting
+and formatting the code, and [Jest](https://jestjs.io/) for testing.
 
 Our pre-commit hooks verify that the linter and tests pass when committing.
 
 ### Scripts
 
-The `package.json` file contains various scripts for common tasks:
-
-- `yarn bootstrap`: setup project by installing all dependencies and pods.
-- `yarn typescript`: type-check files with TypeScript.
-- `yarn lint`: lint files with ESLint.
-- `yarn test`: run unit tests with Jest.
-- `yarn example start`: start the Metro server for the example app.
-- `yarn example android`: run the example app on Android.
-- `yarn example ios`: run the example app on iOS.
+The `package.json` file for the root package contains various scripts for common
+tasks, and the `scripts/` folder for less-common tasks. The scripts in the
+individual packages should remain minimal.
 
```

Comment:
Keeping this up to date is obviously hard (we didn't), and these are almost all self-explanitory